Babylon/commands/api/scenarios/create.py

```diff
@@ -0,0 +1,51 @@
+import json
+from typing import Any
+
+from click import command, option
+
+from Babylon.commands.api.scenarios.service.api import ScenarioService
+from Babylon.utils.credentials import pass_azure_token
+from Babylon.utils.decorators import timing_decorator, wrapcontext, retrieve_state
+from Babylon.utils.response import CommandResponse
+
+payload = json.dumps({
+    "name": "Creating scenario with Babylon",
+    "description": "Brewery master reference analysis",
+    "tags": ["Brewery", "reference"],
+    "runTemplateId": "hundred",
+    "security": {
+        "default": "viewer",
+        "accessControlList": [{
+            "id": "elena.sasova@cosmotech.com",
+            "role": "admin"
+        }],
+    },
+})
+
+
+@command()
+@wrapcontext()
+@retrieve_state
+@pass_azure_token("csm_api")
+@timing_decorator
+@option("--organization_id", "organization_id", type=str)
+@option("--workspace_id", "workspace_id", type=str)
+def create(state: Any, organization_id: str, workspace_id: str, azure_token: str) -> CommandResponse:
```

Comment:
need to add the file to create a scenario, it doesn't come from state
+ add select to allow choosing to save it or not in state

Reply:
1. the select option will be deprecated -> won't do